ui/src/components/Organization/List.tsx

```diff
@@ -9,17 +9,33 @@ interface Props {
   organizations: Organization[];
 }
 
+const defaultBand = (brand: string, name: string) => {
```

Comment:
This is can be a functional component. I differentiate b/w functional components and normal functions by 2 things:
1. Functional components are named in capital case i.e `DefaultBrand` instead of `defaultBrand`
2. Functional components receive a single argument (for most cases) `props` which is an object.

This component can receive the organization itself as its argument, and abstract away how we show its logo. i.e we can have it as `<OrgLogo organization={organiztion} />`